packages/npm-dependencies-validation/test/utils/npmUtil.spec.ts

```diff
@@ -0,0 +1,36 @@
+import { spawnCommand, getNPM } from "../../src/utils/npmUtil";
+import { expect } from "chai";
+
+describe("npmUtil unit test", () => {
+  it("spawnCommand failed", () => {
+    return expect(
+      spawnCommand(["ls"], "./non_existing_path")
+    ).to.be.rejectedWith(`spawn ${getNPM()} ENOENT`);
```

Comment:
should not chai as promised be used with .eventually keyword?

https://github.com/domenic/chai-as-promised

Reply:
not always, this one is also OK